### DISCUSSION TITLE
Serve Isso over main site domain

### DISCUSSION THREAD

**Discussion by Mikeb30** [posativ/isso#777] [Q&A]
Don't think it's that.

I am trying to serve isso over my main domain, blog.mydomain.com
Following this: https://posativ.org/isso/docs/setup/sub-uri/ I still get an error:

`Failed to load resource: the server responded with a status of 404 ()`

**Comment by Konzertheld** [comment]
How do you serve your blog? Please provide your Nginx configuration.
